src/components/Autocomplete/AutocompleteInjected.tsx

```diff
@@ -21,6 +22,8 @@ export default function AutocompleteInjected({ onSubmit }: Props) {
     searchFormRef.current = document.querySelector<HTMLFormElement>(selectors.searchForm)
   }, [])
 
+  const redirect = useNostoAppState(state => state.response?.redirect)
```

Comment:
add to GH preact instructions to always use the most specific selector function for useNostoAppState